apps/backend/src/app/hotel/hotel.service.ts

```diff
@@ -14,27 +15,28 @@ export class HotelService {
     });
   };
   // Get hotel table data
-  async getData(tableName: string): Promise<any> {
+  async getData(tableName: string): Promise<object> {
```

Comment:
object should be Hotel[]

Reply:
yeah I know that, but when I'm try to change it into Hotel[] the error in return appear and I'm not able to resolve it. 
![image](https://github.com/giapnguyen-innomize/training-new-intern/assets/71269151/546f5a5c-e42b-4794-9c1c-c9b8e1cd1df2)